### ISSUE TITLE
How can I clean everything up on simplied UI of nsx manager after kubeadm reset??

### ISSUE THREAD

**Issue by joungwoongpark** [dumlutimuralp/k8s-with-nsx-t-2.5.x#3]
Hi.
Thank for your post.
I finished intergrating with K8S successfully.
But I have a problem after kubectl delete ncp.yml and kubeadm reset.( this means I deleted k8s cluster)

When I deleted segments, IP Pool and IP Block on the Simplied UI of nsx manager.
I found It could not.
along error message. There is someting what still remained at there.
I guess it is coredns. but as I know
coredns would be cleared after kubeadm reset.
So What is the problem??and How can I delete segments, IP Pool and IP Block??

( I used nsxt 2.4.1, k8s 1.15.3 and NCP 2.5.0 and 2.5.1(both same issue))

**Comment by dumlutimuralp** [comment]
@joungwoongpark I see that you opened the same issue again. I replied to your previous issue and @yasensim also provided his insights as well, hence I closed the issue.  Please refer to it to get moe details. It is already here  =>https://github.com/dumlutimuralp/k8s-with-nsx-t-2.5.x/issues/2.  I will close this issue again in 48 hours.

**Comment by joungwoongpark** [comment]
Hi. I tried it as your reply.
but There is still misunderstand. 
Here is my progress.
1. kubectl delete -f ncp-rc.yml
2. python3 nsx_policy_cleanup.py --mgr-ip=70.60.31.249 -u admin -p xxx -c e_test_k8s_cluster --no-warning -r (not working python but working python3)
3. kubeadm reset.

but in progress 2. 
There are two issues.
1. in case of typing python. there is an error message like below.

Traceback (most recent call last):
  File "nsx_policy_cleanup.py", line 23, in <module>
    import requests
ImportError: No module named requests

So. How do I fix it??

2. I tried python3 ..., then it worked. but I cloud not clean everything up on nsx manager UI with tagging "e_test_k8s_cluster". cmd just showed like below and I guess There was no deleting action.
 
70.60.31.249/policy/api/v1/infra/tier-0s
70.60.31.249/policy/api/v1/infra/tier-1s
Found 7 top-tier routers

So, What is the reason??

I always appreciate your assistance.
